tests/ha_master/run.sh

```diff
@@ -133,7 +134,7 @@ function test_list_member() {
 		echo "kill leader" $leader
 		ps aux | grep $leader | awk '{print $2}' | xargs kill || true
 		check_port_offline ${master_ports[$leader_idx]} 20
-		sleep 5
+		check_port_offline ${master_peer_ports[$leader_idx]} 20
```

Comment:
Not know why this port is freed later than ports in previous line 🤔

Reply:
have no idea about this... just add a checker here